addons/l10n_ua/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class PartnerUA(models.Model):
+    _inherit = "res.partner"
+
+    company_registry = fields.Char(
```

Comment:
Same question for this field. What is it use for?

Reply:
Btw, can l10n_xx module make field `name` translatable in models `res.company` and `res.partner`?
Sometimes it is useful to have original company name in Cyrillic and transliterated name for abroad customers.

Reply:
we could do this modules in OCA localization